test/interaction/Debug.out

```diff
@@ -7,8 +7,8 @@
 (agda2-verbose "Wrote interface file. ")
 (agda2-verbose "Accumulated statistics. ")
 (agda2-verbose "Finished Debug. ")
-(agda2-verbose " current options = [--safe: False, --without-K: False, --no-universe-polymorphism: False, --no-cumulativity: True, --cubical/--erased-cubical: False, --guarded: False, --prop: False, --two-level: False, --rewriting: False, --sized-types: False, --guardedness: False] ")
-(agda2-verbose " imported options = [--safe: False, --without-K: False, --no-universe-polymorphism: False, --no-cumulativity: True, --cubical/--erased-cubical: False, --guarded: False, --prop: False, --two-level: False, --rewriting: False, --sized-types: False, --guardedness: False] ")
+(agda2-verbose " current options = [--safe: False, --without-K: False, --cubical-compatible: False, --no-universe-polymorphism: False, --no-cumulativity: True, --cubical/--erased-cubical: False, --guarded: False, --prop: False, --two-level: False, --rewriting: False, --sized-types: False, --guardedness: False] ")
```

Comment:
@UlfNorell : It is weird that such a patch appears in this PR (and not in #5862, but see https://github.com/agda/agda/pull/5862#issuecomment-1108236780).
But besides that, should `current options` report _both_ `without-K` and `cubical-compatible`?  I would have expected the latter only...